### PR TITLE
Feat: More Recovery code endpoints

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Adds new endpoints to have more control over recovery codes.

- [x] Users Service
- [x] Account service

## Test Plan

- [x] Manual QA

## Related PRs and Issues

x

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -17,7 +17,6 @@ tasks:
 
 ports:
   - port: 8080
-    onOpen: open-preview
```
Comment (Meldiron) [inline .gitpod.yml:20]: On purpose. If server loads while you are outside of Chrome window, and this preview opens, it crashes whole chrome. Happened multiple times, very annoying.

**Comment by Meldiron** [comment]
Did basic manual QA. Will do more with Console changes and eventually we will add tests.

File: src/Appwrite/Auth/Auth.php [hunk 2]
```diff
@@ -86,6 +86,11 @@ class Auth
     public const TOKEN_LENGTH_OAUTH2 = 64;
     public const TOKEN_LENGTH_SESSION = 256;
 
+    /**
+     * MFA
+     */
+    public const MFA_RECENT_DURATION = 600; // 5 minutes
```
Comment (Meldiron) [inline src/Appwrite/Auth/Auth.php:92]: 15 mins?

**Review by Meldiron — COMMENTED** [review]
✅

File: app/controllers/api/account.php [hunk 3]
```diff
@@ -3743,6 +3753,77 @@
...
+        }
+
+        $mfaRecoveryCodes = Type::generateBackupCodes();
+        $user->setAttribute('mfaRecoveryCodes', $mfaRecoveryCodes);
+        $dbForProject->updateDocument('users', $user->getId(), $user);
+
+        $queueForEvents->setParam('userId', $user->getId());
+
+        $document = new Document([
+        'recoveryCodes' => $mfaRecoveryCodes
```
Comment (TorstenDittmann) [inline app/controllers/api/account.php:3791]: check linter

File: app/controllers/api/users.php [hunk 4]
```diff
@@ -1591,6 +1591,132 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
...
+        }
+
+        $mfaRecoveryCodes = Type::generateBackupCodes();
+        $user->setAttribute('mfaRecoveryCodes', $mfaRecoveryCodes);
+        $dbForProject->updateDocument('users', $user->getId(), $user);
+
+        $queueForEvents->setParam('userId', $user->getId());
+
+        $document = new Document([
+        'recoveryCodes' => $mfaRecoveryCodes
```
Comment (TorstenDittmann) [inline app/controllers/api/users.php:1714]: linter

**Pull request merged**